test/snapp/snapp_base.js

```diff
@@ -30,11 +30,11 @@ contract("SnappBase", async accounts => {
   beforeEach(async () => {
     const lib1 = await IdToAddressBiMap.new()
 
-    await SnappBaseCore.link(IdToAddressBiMap, lib1.address)
+    await SnappBaseCore.link("IdToAddressBiMap", lib1.address)
```

Comment:
Why all these changes?

Reply:
I tried to explain it in the comment:

>I needed to use the following linking method in tests:
await BatchExchange.link("IdToAddressBiMap", lib1.address)
instead of
await BatchExchange.link(IdToAddressBiMap, lib1.address)
as otherwise, I would get:
Error: Cannot link the contract without an address. Cf. https://travis-ci.org/gnosis/dex-contracts/builds/642488383?utm_source=github_status&utm_medium=notification

Reply:
But why? What difference does it make? I'd like to understand these changes better.